### PR TITLE
Update projectile textures

### PR DESCRIPTION
**Pull request by LordDeatHunter**

### CODE REVIEW

File: assets/texture/ally_bullet.png
Comment (cdsupina) [inline assets/texture/ally_bullet.png:1]: I'm a little concerned that the color of the allied bullet you chose won't contrast enough with the backgrounds. Could you adjust the colors of the allied bullet and associated animations to contrast a bit more with the dark space backgrounds?

**Review by cdsupina — CHANGES_REQUESTED** [review]
I just had one concern about color contrast with the allied bullet. Great work again though!

File: assets/texture/ally_bullet.png
Comment (LordDeatHunter) [reply assets/texture/ally_bullet.png:1]: I made all the bullets lighter 👌

**Review by cdsupina — APPROVED** [review]
LGTM!

**Review by varoonp123 — APPROVED** [review]

**Pull request merged**